setup.cfg

```diff
@@ -45,7 +45,8 @@ python_version = 3.8
 strict_optional = True
 plugins = numpy.typing.mypy_plugin
 exclude = examples/models.*
-disallow_untyped_defs=true
+disallow_untyped_defs = true
+ignore_missing_imports = True
```

Comment:
I suppose `ignore_missing_imports` has been set to `true` to hide this warning:

```
black_it/samplers/gaussian_process.py:26: error: Cannot find implementation or library stub for module named "sklearn.gaussian_process"  [import]
black_it/samplers/gaussian_process.py:26: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
```

It would be more explicit to not activate `ignore_missing_imports` project-wise, and eventually selectively silence the warning via `# type: ignore[import]` (documentation reference: https://mypy.readthedocs.io/en/stable/error_codes.html#silencing-errors-based-on-error-codes).

However, since the mypy report is already unclean, in this case I'd just keep the warning on (without adding `# type: ignore[import]`), so that we could fix the error in a future PR.

Reply:
addressed in e8ae4c6ad65f (to be squashed)